gitlab-runner/values-okteto.yaml

```diff
@@ -0,0 +1,43 @@
+gitlab-runner:
+  
+  ## The GitLab Server URL (with protocol) that want to register the runner against
+  ## ref: https://docs.gitlab.com/runner/commands/README.html#gitlab-runner-register
+  gitlabUrl: https://gitlab.com/
+  
+  ## The registration token for adding new Runners to the GitLab server. This must
+  ## be retrieved from your GitLab instance.
+  ## ref: https://docs.gitlab.com/ee/ci/runners/
+  runnerRegistrationToken:
+  
+  ## Configure the maximum number of concurrent jobs
+  concurrent: 1
+
+  ## Defines in seconds how often to check GitLab for a new builds
+  checkInterval: 30
+
+  ## Configuration for the Pods that the runner launches for each new job
+  runners:
+    image: ubuntu:18.04
+
+    builds:
+      cpuLimit: 100m
```

Comment:
Is this enough for CI jobs?

Reply:
That's completely dependend on the CI job, so i don't know. It was enough to run tests and build a go binary. It's exposed in values-okteto.yml so the caller can customize it based on their expected load.

If you have a suggested value to put, happy to change it. Maybe 0.5 cpu and 512mb of memory?

Reply:
I would use our default, 0.5 cpu and 1Gb, but whatever you think works better